packages/l/libelf/xmake.lua

```diff
@@ -34,6 +33,9 @@ package("libelf")
             package:add("defines", "__libelf_i64_t=int64_t")
         end
         io.replace("./configure", "main(){return(0);}", "int main(){return(0);}", {plain = true})
+
+        os.vrunv("autoreconf", {"-fiv"})
```

Comment:
xmake will call it if configure is missing. https://github.com/xmake-io/xmake/blob/2774ecbc0961e1e2b1f995d8937b44c2c90e11c6/xmake/modules/package/tools/autoconf.lua#L518

you can remove ./configure to force to regenerate it.